pkg/client/cache/fifo.go

```diff
@@ -241,6 +241,12 @@ func (f *FIFO) Replace(list []interface{}, resourceVersion string) error {
 	return nil
 }
 
+// Resync will touch all objects to put them into the processing queue
+func (f *FIFO) Resync() error {
+	// Nothing to do
+	return nil
```

Comment:
Yeah - I probably wasn't clear enough.
We basically want to do a similar thing we are doing in Add. So we are iterating over all elements that exists, and add to queue only those that are not in it already.

Does it sound ok for you?